src/styles/block-card.css

```diff
@@ -0,0 +1,90 @@
+/* #region Card */
+.card {
+  display: flex;
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-direction: column;
+}
+
+.card__box-img {
+  display: flex;
+  padding: 0 3px;
+  margin: 0 0 40px;
+}
+
+.card__img {
+  width: 100%;
+  height: 134px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 0 4px;
+}
+
+.card__code,
+.card__reviews,
+.card__text-price {
+  font-weight: 400;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-text-code-price);
+  margin: 0 0 16px;
+}
+
+.card__evaluation {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 0 24px;
+}
+
+.card__reviews {
+  font-size: 10px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 0 16px;
+}
+
+.card__text-price {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-text-code-price);
+}
+
+.card__digit-price {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  height: var(--button-height);
+  color: var(--button-color-text-bgc-hover);
+  background-color: var(--button-color);
+  box-sizing: border-box;
```

Comment:
you can just move `box-sizing: border-box` to `*` selector and no need to repeat it then